scripts/collections.cdc

```diff
@@ -124,6 +125,24 @@ pub fun main(address: Address) : {String : MetadataCollection}? {
 	}
 	*/
 
+	let rareRoomCollection = account.getCapability<&RareRooms_NFT.Collection{RareRooms_NFT.RareRooms_NFTCollectionPublic}>(RareRooms_NFT.CollectionPublicPath).borrow()!
```

Comment:
you cannot ! this here, that will throw an exception if a user does not have a collection. Just remove it here and ! it inside the if. Or use the pattern with check we do in the other examples.